t/routes-application.t

```diff
@@ -115,23 +116,37 @@ subtest 'Get output variables' => {
     }
 }
 
-# TODO: failing on GitHub actions, ok locally.
-#subtest 'Get excel export' => {
-#    test-service routes($fake-store), :$fake-auth, {
-#        test-given '/get_excel_export', {
-#            test post(
-#                content-type => 'application/x-www-form-urlencoded',
-#                body => {
-#                    :datasetName('TestSingle'),
-#                    :language('de'),
-#                    :!withFilters
-#                }),
-#                status => 200;
-#        };
-#        check-mock $fake-store,
-#            *.called('get-excel-export', times => 1);
-#    }
-#}
+subtest 'Get excel export' => {
+    test-service routes($fake-store), :$fake-auth, {
+        test-given '/export/excel', {
+            test post(
+                    content-type => 'application/x-www-form-urlencoded',
+                    body => {
+                        :datasetName('TestSingle'),
+                        :language('de'),
+                    }),
+                    status => 200;
```

Comment:
Maybe worth testing the content type of the response too?

Reply:
How would I do that? And doesn't OpenAPI enforce that anyway?

Reply:
> How would I do that?

Pass `content-type => '...'` after `status => 200`. [Docs](https://cro.services/docs/reference/cro-http-test#content-type). 

> And doesn't OpenAPI enforce that anyway?

Good point, yes, so it is a bit surplus to requirements. So let's not worry about it.